tests/gateway_with_fqdn/gateway_fqdn_test.go

```diff
@@ -0,0 +1,47 @@
+package test
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/threefoldtech/terraform-provider-grid/tests"
+)
+
+func TestSingleNodeDeployment(t *testing.T) {
+	/* Test case for deployeng a gateway with fdqn.
+
+	   **Test Scenario**
+
+	   - Deploy a gateway with fdqn.
+	   - Check that the outputs not empty.
+	   - Check that ygg ip is reachable.
+	   - Check that gateway point to backend.
+	   - Destroy the deployment
+	*/
+
+	// retryable errors in terraform testing.
+	name := tests.RandomName()
+	backend := "http://69.164.223.208:443"
+	fdqn := "remote." + name + ".grid.tf"
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./",
+		Vars: map[string]interface{}{
+			"fdqn":    fdqn,
```

Comment:
can u add those vars to readme run tests?

Reply:
also all other needed env vars in all tests

Reply:
fdqn not a env vars